django/db/migrations/serializer.py

```diff
@@ -270,6 +272,40 @@ def serialize(self):
         return "uuid.%s" % repr(self.value), {"import uuid"}
 
 
+class ClassSerializer(BaseSerializer):
+    def serialize(self):
+        _module = self.value.__module__
+        module_first_part, module_second_part = _module.split('.')
+        _class = self.value.__class__
+        _path = ""
+        if module_second_part.startswith('_'):
+            related_modules = [
+                m for m in sys.modules
+                if module_first_part in m and
+                m != _module and len(m.split('.')) > 1 and not
+                m.split('.')[1].startswith('_')
+            ]
```

Comment:
Iterating over `sys.modules` is going to be significantly slow for any medium sized Django projects.